translated/en_us/sidebar_translations.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
It doesnt make sense for this file to be in /translated. As far as im concerned that entire directory shouldnt be touched by humans. I would maybe move this into the root, or alongside the sidebar ts files.